core/src/main/java/com/rim/IceField/Iceberg.java

```diff
@@ -99,9 +108,20 @@ public void setIsStable(boolean b) {
     }
 
     public void Add_currentPlayers(PlayerBase p) {
-        currentPlayers.add(p);
-        p.setCurrentIceberg(this);
-      //  System.out.println("Player" + p.ID + " was added to the iceberg number: " + num);
+        if(currentPlayers.size() < maxNumOfPlayers) {
```

Comment:
sorry, I am out of the context, what is this method for?

Reply:
This method is called when a player moves to another iceberg. It adds another player to the list of players that are already on the iceberg. We need to check if when we add new players they don't exceed the maximum number of players that can stay on an iceberg before it breaks,

Reply:
I see, thanks!